### PR TITLE
#144937429 Setup Base Project

### PR DESCRIPTION
**Pull request by efeguono**
#### What does this PR do?
It sets up the base project requirements

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://Eguono:Daniella12345&@ds133271.mlab.com:33271/kiba-cfh
```
Comment (q-ode) [inline .env]: This is still here. Please remove.

File: package.json [hunk 2]
```diff
@@ -9,40 +9,41 @@
     "url": "https://github.com/cardsforhumanity/cardsforhumanity.git"
   },
   "engines": {
-    "node": "0.10.x",
-    "npm": "1.2.x"
+    "node": "6.10.x",
+    "npm": "3.10mongoo.x"
```
Comment (q-ode) [inline package.json]: What's the purpose of this please?

File: server.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ var express = require('express'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
-
+require('dotenv').config();
```
Comment (q-ode) [inline server.js]: Nitpick: Please move this to the line just before the set of `require` statements assigned to variables.

**Review by q-ode — CHANGES_REQUESTED** [review]
Hi Efe,

Please address the comments so we can unblock others.

File: package.json (hunk 2 above)
Comment (efeguono) [reply package.json]: This is required for deploying the application to Heroku, It specifies the version of npm and node that runs on heroku

Comment (q-ode) [reply package.json]: This however has nothing to do with this task. I remember there's a chore specifically for deploying to Heroku.

**Pull request closed without merging**